### PR TITLE
Increase default XHR timeout and make it configurable

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
The default timeout for XHR uploads has been increased from 30 to 60 seconds. Also, it can now be configured via the `config.json` file (in ms).

## Related Issue

- Fixes https://github.com/owncloud/web/issues/7900

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

### CODE REVIEW

File: packages/web-runtime/src/composables/upload/useUpload.ts [hunk 1]
```diff
@@ -104,7 +104,8 @@ export function useUpload(options: UploadOptions): UploadResult {
       ...(isTusSupported && {
         tusMaxChunkSize: unref(tusMaxChunkSize),
         tusHttpMethodOverride: unref(tusHttpMethodOverride),
-        tusExtension: unref(tusExtension)
+        tusExtension: unref(tusExtension),
+        xhrTimeout: store.getters.configuration?.options?.upload?.xhr?.timeout || 60000
```
Comment (kulmann) [inline packages/web-runtime/src/composables/upload/useUpload.ts:108]: huh, why do you set the xhr timeout if tus is supported?

**Review by kulmann — CHANGES_REQUESTED** [review]
Please add a short description of the new option to the `getting started` page in the `docs` folder (there is a section for options).

File: packages/web-runtime/src/composables/upload/useUpload.ts (hunk 1 above)
Comment (JammingBen) [reply packages/web-runtime/src/composables/upload/useUpload.ts:108]: Arrrghs 😅

**Review by kulmann — APPROVED** [review]
Nice! 👍

**Pull request merged**